client/js/config/Theme/index.js

```diff
@@ -67,16 +77,16 @@ const THEME = {
     },
   },
   padding: {
-    default: int => {
-      return 16 * int;
-    },
     accountScreen: {
       paddingHorizontal: 32,
     },
     inputField: {
       paddingHorizontal: 16,
       paddingVertical: 12,
     },
+    bottomGap: {
+      paddingBottom: 48,
+    },
```

Comment:
avoid using huge margins and paddings. Use flexbox.

Reply:
This is the extra margin for the scroll view.
So when a user scrolls to the bottom they can feel more comfortable with the extra space.